recipe/meta.yaml

```diff
@@ -30,43 +30,32 @@ requirements:
   run:
     - python
 
-{% set tests_to_skip = "_not_a_real_test" %}
-# Seems to fail with current version of blas for large numbers
-# https://github.com/conda-forge/numpy-feedstock/pull/179#issuecomment-569591828
-{% set tests_to_skip = tests_to_skip + " or test_sincos_float32" %}  # [linux64]
-# Likely bugs in openblas????
-# https://github.com/numpy/numpy/issues/14438#issuecomment-560940664
-{% set tests_to_skip = tests_to_skip + " or test_einsum_sums_cfloat64" %}  # [ppc64le]
-# Only the complex256 system is failing, but I don't know how to skip it on its own
-# # https://github.com/numpy/numpy/issues/15243
-{% set tests_to_skip = tests_to_skip + " or test_loss_of_precision" %}     # [ppc64le]
-# Probably needs too much memory on ppc64le, it sometimes fails on travis
-{% set tests_to_skip = tests_to_skip + " or test_large_zip" %}   # [ppc64le]
-# Extra slow tests on aarch64 and ppc64le
-{% set tests_to_skip = tests_to_skip + " or test_may_share_memory_easy_fuzz" %}   # [ppc64le or aarch64]
-{% set tests_to_skip = tests_to_skip + " or test_may_share_memory_harder_fuzz" %}   # [ppc64le or aarch64]
-{% set tests_to_skip = tests_to_skip + " or test_unary_ufunc_call_fuzz" %}   # [ppc64le or aarch64]
-{% set tests_to_skip = tests_to_skip + " or test_count_nonzero_all" %}   # [ppc64le or aarch64]
-{% set tests_to_skip = tests_to_skip + " or test_diophantine_fuzz" %}   # [ppc64le or aarch64]
-{% set tests_to_skip = tests_to_skip + " or test_generalized_sq_cases" %}   # [ppc64le or aarch64]
-{% set tests_to_skip = tests_to_skip + " or test_may_share_memory_harder_fuzz" %}   # [ppc64le or aarch64]
-{% set tests_to_skip = tests_to_skip + " or test_large_zip" %}   # [ppc64le or aarch64]
-
 test:
   requires:
     - pytest
     - pytest-xdist  # [aarch64]
     - hypothesis
     - setuptools
+    # make sure we test with all blas implementations
+    - blas * *{{ blas_impl }}
+  files:
+    - run_test_suite.py
   commands:
     - f2py -h
-    - export OPENBLAS_NUM_THREADS=1  # [unix]
-    - set OPENBLAS_NUM_THREADS=1  # [win]
-    - pytest --verbose --pyargs numpy -k "not ({{ tests_to_skip }})" --durations=0         # [not aarch64]
```

Comment:
This could have been
`pypy3 --jit off -m pytest <rest>`

Reply:
That's true, but I actually wanted to get closer to the setup that the scipy-feedstock has, because all the different switches are IMO easier to follow in a `run_test.py`, rather than in the YAML with variables & selectors.

Reply:
> because all the different switches are IMO easier to follow in a run_test.py, rather than in the YAML with variables & selectors

Well, since you've removed all the switches, it is easier to read, but I doubt it will be easier if you ported the skips.

Reply:
It's a fair point. I only needed to readd two skips, which also skews the perception. But this is just a WIP PR anyway - right now those changes are what I wanted to try; if desirable, I can carve them out into a separate PR, if not, no biggie.